Tensile/TensileRetuneLibrary.py

```diff
@@ -115,11 +133,21 @@ def runBenchmarking(solutions, problemSizes, outPath):
     forBenchmark = True
     # TODO make this work with TileAware selection
     returncode = ClientWriter.runClient(libraryLogicPath, forBenchmark, False)
-
     if returncode:
-        printExit("BenchmarkProblems: Benchmark Process exited with code %u" % returncode)
+        printWarning("BenchmarkProblems: Benchmark Process exited with code {}".format(returncode))
```

Comment:
Language? With this change you are not really exiting here.

Reply:
I've updated this statement to more accurately describe the state of the program

Reply:
@benjaminulmer Please don't forget to re-request a review by @wbgilmartin, thanks.